### DISCUSSION TITLE
Where to put files for a statically generated root folder?

### DISCUSSION THREAD

**Discussion by tyeeman** [datenstrom/community#380] [Ask a question] [answered]
Where should I place a folder (with files in it) if I want it to be located in the static generated root folder?

**Comment by tyeeman** [comment]
I added a folder to the media folder and it got copied over when the static site was built but I would like it to be in the root if possible. It's for an admin folder btw.

**Comment by markseuffert** [comment]
I think the [media folder](https://datenstrom.se/yellow/help/how-to-change-the-media) is the best place for additional folders. Datenstrom Yellow creates a static website from your content + media + system folders, other folders are not included at the moment. 

I am curious, what do you use the admin folder for?

**Comment by tyeeman** [comment]
Haha, Netlify again! They have their Netlify CMS which is sort of cool, very small size files but seems to work fine. I have 3 files totalling 2kbyte, config.yml, index.html, netlify.toml and that's it. 

![2018-11-27_100000](https://user-images.githubusercontent.com/4469306/49101539-4cc08100-f22b-11e8-9005-6f3933af881d.png)

**Comment by markseuffert** [comment]
Thanks for the explanation. I am sorry, the current software doesn't support this feature.

**Comment by markseuffert** [comment]
If someone would like to see this feature, please add +1 to the top of this issue.

**Comment by markseuffert** [comment, marked as answer]
Fixed it. Now your `admin` folder should be copied over to the statically generated website. 

Please update your website and let me know if this works for you.

**Comment by markseuffert** [comment]
Give me a shout if this isn't working as expected. Feedback is always welcome.
